clock.c

```diff
@@ -163,6 +163,9 @@ static int clock_alttime_offset_append(struct clock *c, int key, struct ptp_mess
 	struct tlv_extra *extra;
 	int tlv_len;
 
+	int a = 2;
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Maintainability$</h3>

Consider explaining the purpose of the dummy variables or remove them if they are not needed for production code.